cmd/xcode.go

```diff
@@ -108,6 +114,67 @@ func scanXcodeProject(_ *cobra.Command, _ []string) error {
 		xcodeCmd.SDK = paramXcodebuildSDK
 	}
 
+	if paramXcodeDestination != "" {
+		xcodeCmd.DESTINATION = paramXcodeDestination
+	} else {
+		var project xcodeproj.XcodeProj
```

Comment:
I suggest moving the destination scanning to the `utility` package (so that we do not bloat this file + this file mostly contains glue code)